requirements/edx/base.in

```diff
@@ -64,7 +63,9 @@ django-storages==1.4.1
 django-user-tasks
 django-waffle==0.12.0
 django-webpack-loader               # Used to wire webpack bundles into the django asset pipeline
+djangorestframework==3.7.7
```

Comment:
Why the hard version pin?

Reply:
Because I wanted to upgrade enough to get drf-yasg, and not have to swallow the entire upgrade all at once, for now.